pkg/azure/types/types.go

```diff
@@ -145,6 +146,13 @@ func (a *AzureInterface) extractIDs() {
 		if len(segs) >= 11 {
 			a.vmID = segs[10]
 		}
+	// Interface from a standalone instance:
+	// //subscriptions/xxx/resourceGroups/yyy/providers/Microsoft.Network/networkInterfaces/iii
+	case strings.Contains(a.ID, "/Microsoft.Network/"):
```

Comment:
Mind adding a couple of test cases for this via https://github.com/cilium/cilium/blob/master/pkg/azure/types/types_test.go#L75?

Reply:
Done